core/platform/app_identity/gae_app_identity_services.py

```diff
@@ -38,6 +38,30 @@ def get_application_id():
     return app_identity.get_application_id()
 
 
+def get_access_token(scopes):
+    """The OAuth 2.0 access token to act on behalf of the application.
+
+    This token will be cached.
+
+    A token will be generated and cached for the service account for the App
+    Engine application.
+
+    Each application has an associated Google account. This function returns an
+    OAuth 2.0 access token that corresponds to the running app. Access tokens
+    are safe to cache and reuse until their expiry time as returned. This method
+    caches access tokens using both an in-process cache and memcache.
+
+    Args:
+        scopes: str | list(str). The requested API scope as a string, or a list
```

Comment:
Make it just str, for our purposes, to keep the API narrow. (Maybe call it `scope` here?)

Reply:
Done.